lxc/profile.go

```diff
@@ -60,16 +60,18 @@ lxc profile edit <profile>
     Example: lxc profile edit <profile> # launch editor
              cat profile.yml | lxc profile edit <profile> # read from profile.yml
 
-lxc profile apply <container> <profiles>
-    Apply a comma-separated list of profiles to a container, in order.
+lxc profile assign <container> <profiles>
+    Assign a comma-separated list of profiles to a container, in order.
     All profiles passed in this call (and only those) will be applied
-    to the specified container.
+    to the specified container, i.e. it sets the list of profiles exactly to
+    those specified in this command. To add/remove a particular profile from a
+    container, use {add|remove} below.
     Example: lxc profile apply foo default,bar # Apply default and bar
```

Comment:
On Mon, May 23, 2016 at 06:04:39AM -0700, ctrlrsf wrote:
> >      All profiles passed in this call (and only those) will be applied
> > -    to the specified container.
> > +    to the specified container, i.e. it sets the list of profiles exactly to
> > +    those specified in this command. To add/remove a particular profile from a
> > +    container, use {add|remove} below.
> >      Example: lxc profile apply foo default,bar # Apply default and bar
> 
> I think the lxc profile apply examples should also be renamed to lxc profile assign for consistency.

Good point, I'll change that.

> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/lxc/lxd/pull/2038/files/3a6d6a56d214f7b212d283f6c7ba992ef99e618f#r64218196